### PR TITLE
BRS-898: review data page barebones

### PR DESCRIPTION
**Pull request by cameronpettit**
### Jira Ticket:
BRS-898

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-898

### Description:
This ticket implements the barebones UI for variance checking and data review. 

The first iteration of the API requires that a `subarea` and `activity` are provided, so those fields are mandatory in the UI. The `park` field is mandatory as it is used to filter down the number of subareas shown in the subarea typeahead list. The `date` field is optional.

Upon fetching a list of variances based on the filters, the page is populated with accordions. By clicking on 'View', you can navigate directly to the record with the variance in question.

### CODE REVIEW

File: src/app/services/keycloak.service.ts [hunk 1]
```diff
@@ -153,9 +153,27 @@ export class KeycloakService {
...
+      'review-data'
+    ]
+
+    if (this.configService['configuration'].ENVIRONMENT === 'prod') {
+      // we are in prod
+      if (blockedInProd.find(route => route === service)) {
+        return false;
+      }
+    }
+
```
Comment (marklise) [inline src/app/services/keycloak.service.ts:176]: We should probably set a specific env var like `this.configService['configuration'].IS_FEATURE_X_ENABLED === 'prod'` vs. checking against the specific environment. Won't require a new build as a result.

**Review by marklise — APPROVED** [review]
/w nit above.

File: src/app/services/keycloak.service.ts (hunk 1 above)
Comment (cameronpettit) [reply src/app/services/keycloak.service.ts]: Removed this env checker in `4c7dc4d`. Can toggle features with a feature flag in the future if necessary.

**Review by meredithom — APPROVED** [review]
/w nit

console error when selecting sub-area (present on both data review and data entry pages):
image

**Pull request merged**